state/execution.go

```diff
@@ -238,6 +238,20 @@ func (blockExec *BlockExecutor) ApplyBlock(
 
 	fail.Fail() // XXX
 
+	blockGenerationTimeMs := float64((time.Now().UnixNano()-execStartTime)/1000000) / 1000.0
```

Comment:
`blockGenerationTimeMs` seems to measure the time of the `ApplyBlock`, but the same value is already measured in metric by `CommitExecuting+CommitCommissioning+CommitRechecking`.

Reply:
Oh, I didn't know about those variables. I'll pull this PR and update logging with those existing variables.

Thanks for pointing it out.